README.md

```diff
@@ -25,7 +25,14 @@ dependencies {
 ```
 
   * [Gson Download](https://maven-badges.herokuapp.com/maven-central/com.google.code.gson/gson) downloads at Maven Central
-  * For Maven check "Dependency Information" tab, on the left side.
+  * To use Gson with Maven
+  ```
```

Comment:
nit: you can add `xml` after the three backticks to get highlighting, fwiw (just green text on the tags).